app/models/user.rb

```diff
@@ -15,8 +15,12 @@ class User < ActiveRecord::Base
   
   has_attached_file :avatar, styles: {thumb: '100x100>',
                                       square: '200x200#',
-                                      medium: '300x300>'
-                                      }
+                                      medium: '300x300>'}, 
+                              default_url: "************",
+                              storage: :s3,
+                              bucket: ENV['bucket'],
+                              s3_credentials: { access_key_id: ENV["access_key_id"],
```

Comment:
Line is too long. [84/80]<br>Align the elements of a hash literal if they span more than one line.